src/components/Table/utils/TableState.ts

```diff
@@ -29,11 +29,16 @@ export type SelectedState = "checked" | "unchecked" | "partial";
  */
 export class TableState {
   // A set of just row ids, i.e. not row.kind+row.id
-  private readonly collapsedRows = new ObservableSet<string>();
+  private readonly collapsedRows = new ObservableSet<string>([]);
   private persistCollapse: string | undefined;
-  private readonly selectedRows = new ObservableMap<string, SelectedState>();
+  private readonly rowSelectedState = new ObservableMap<string, SelectedState>();
```

Comment:
Maybe in-source comment that this will have any selected-or-used-to-be-selected rows + any kept-selected rows.